### PR TITLE
Add Dracula Theme

### PR DESCRIPTION
**Pull request by Teyler7**
Hello there,

Please let me know if there is anything I can update with my PR. This is one of my first PR's to open source. I appreciate everything you do 👍 

Thanks!

### CODE REVIEW

File: _posts/theme/2021-08-24-Dracula.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: theme
+title: Dracula
+category: theme
+homepage: https://github.com/Teyler7/dracula-typora-theme
+download: https://github.com/Teyler7/dracula-typora-theme/archive/refs/heads/main.zip
+built-in: false
+author: Teyler7
+thumbnail: dracula-typora.png
```
Comment (abnerlee) [inline _posts/theme/2021-08-24-Dracula.md:9]: This file seems not exist

Comment (Teyler7) [reply _posts/theme/2021-08-24-Dracula.md:9]: @abnerlee My apologies. Thank you for calling this out. I have updated my file path for my thumbnail image.

**Comment by abnerlee** [comment]
I have merged this, but better to use a real thumbnail, so user can see how it looks before opening the detail page.

**Comment by Teyler7** [comment]
@abnerlee Sounds good. I will create a new PR in the future updating my thumbnail. That make sense 👍  Thanks!

**Pull request merged**